browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -39,12 +40,15 @@
 #endif
 
 #if BUILDFLAG(BRAVE_WALLET_ENABLED) && !defined(OS_ANDROID)
-#include "brave/browser/ui/webui/brave_wallet/deprecated_brave_wallet_ui.h"
 #include "brave/browser/ui/webui/brave_wallet/wallet_page_ui.h"
 #include "brave/browser/ui/webui/brave_wallet/wallet_panel_ui.h"
+#include "brave/browser/ui/webui/ethereum_remote_client/ethereum_remote_client_ui.h"
 #include "brave/components/brave_wallet/browser/brave_wallet_utils.h"
 #endif
 
+#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)
```

Comment:
empty?

Reply:
I think https://github.com/brave/brave-core/pull/8941/files#diff-739b827f200537078f68db9cca1601599dac34e7b4077573ba198a925365df27R45 was supposed to go here?

Reply:
Fixed.